backend/prompt_studio/prompt_studio_output_manager/models.py

```diff
@@ -24,6 +24,9 @@ class PromptStudioOutputManager(BaseModel):
     context = models.TextField(
         db_comment="Field to store chunks used", editable=True, null=True, blank=True
     )
+    challenge_data = models.JSONField(
```

Comment:
@hari-kuriakose @Deepak-Kesavan @tahierhussain 
When do we prioritize moving such data to the cloud repo?

Reply:
@chandrasekharan-zipstack We should plan creating a table for keeping all cloud related fields.